akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala

```diff
@@ -54,6 +54,8 @@ object OverflowStrategy {
    */
   def dropBuffer: OverflowStrategy = DropBuffer
 
+  // FIXME shouldn't there be a strategy that drops the new element?
```

Comment:
Yes, definitely! I thought that was dropTail, but I was mistaken, please add a ticket.